.github/workflows/release.yml

```diff
@@ -0,0 +1,80 @@
+name: Release
+
+on:
+  push:
+    branches: [main]
```

Comment:
IMO we should release a version manually, and don't release for every commit (i think it's the common practice)